frontend/src/app/main/data/workspace/modifiers.cljs

```diff
@@ -348,9 +348,12 @@
 ;; Rotation use different algorithm to calculate children modifiers (and do not use child constraints).
 (defn set-rotation-modifiers
   ([angle shapes]
-   (set-rotation-modifiers angle shapes (-> shapes gsh/selection-rect gsh/center-selrect)))
+   (set-rotation-modifiers angle shapes {}))
 
-  ([angle shapes center]
+  ([angle shapes {:keys [as-delta?] :or {as-delta? false}}]
+   (set-rotation-modifiers angle shapes (-> shapes gsh/selection-rect gsh/center-selrect) {:as-delta? as-delta?}))
```

Comment:
In case of changing rotation of several shapes, ¿do we need to do it relative to the selection center? ¿Or should each shape be rotated from its own center?

Reply:
I think in this case it makes more sense to rotate each shape from it's own center but I'll speak about it with UX/UI. Thanks!